packages/contentful--create-contentful-app/src/template.ts

```diff
@@ -4,29 +4,101 @@ import degit from 'degit';
 import rimraf from 'rimraf';
 
 import { CLIOptions, ContentfulExample } from './types';
-import { highlight, warn } from './logger';
+import { highlight, success, warn, wrapInBlanks } from './logger';
 import { rmIfExists } from './utils';
+import fetch from 'node-fetch';
+import inquirer from 'inquirer';
+import chalk from 'chalk';
 
 const EXAMPLES_PATH = 'contentful/apps/examples/';
 
 function isContentfulTemplate(url: string) {
   return Object.values(ContentfulExample).some((t) => url.includes(EXAMPLES_PATH + t));
 }
 
-function makeContentfulExampleSource(options: CLIOptions) {
+async function getGithubFolderNames(username: string, repo: string, path: string) {
+  const url = `https://api.github.com/repos/${username}/${repo}/contents/${path}`;
+
+  const response = await fetch(url);
+  const contents: any = await response.json();
+
+  return contents.filter((content: any) => content.type === "dir")
+  .map((content: any) => content.name);
+}
+  
+  
+async function promptExampleSelection(): Promise<string> {
+  let template = 'typescript'
+
+  // ask user whether to start with an empty template or use an example template
+  const { starter } = await inquirer.prompt([
+    {
+      name: 'starter',
+      message: 'Do you want to start from an example template or use the empty template?',
+      type: 'list',
+      choices: ['empty', 'template'],
```

Comment:
...and here we give choice of empty (what?) or template (empty template or example template?). Feels like we need to change choices to empty and example. And I would use 'blank' instead of 'empty'.

Reply:
Makes sense, I've changed it to "example" and "blank"